### DISCUSSION TITLE
Discord Server / Channel

### DISCUSSION THREAD

**Discussion by fabse-hack** [miguelgrinberg/microdot#196] [Ideas]
Hallo,

do we have an discord server / channel?
Maybe is a good idea for the community arround microdot ?

**Comment by miguelgrinberg** [comment]
The problem I have with discord support is that it isn't indexed by search engines, so all that happens in the channel is not accessible to people that may have similar issues when they search. I would prefer questions and answers to be recorded here or maybe Stack Overflow, so that they are more easily found later on.

**Reply by marcindulak** [reply]
Thank you for this decision.

There are blog posts that describe issues with discord or similar external services
https://blog.mattcen.com/2023/07/04/stop-using-discord-for-your-open-source-communities/

In that regard, even github discussions are not optimal, since they don't auto backlink to github issues, and contribute to the information fragmentation https://github.com/orgs/community/discussions/2971.

**Reply by fabse-hack** [reply]
hmmm in discord can set up an support area where people can open tickets. Is indexable and searchable... good example is the python discord server, they make a really good setup.
